src/command-line/index.js

```diff
@@ -7,7 +7,7 @@ var fsextra = require("fs-extra");
 var path = require("path");
 var Helper = require("../helper");
 
-program.version(pkg.version, "-v, --version");
+program.version(Helper.getVersion(), "-v, --version");
```

Comment:
What do you mean by npm version? All versions are (so far) given git tags as well (which is what git-describe uses to produce these refs).

> As a user, what does that mean, am I running the pre.6 version or a specific commit? Is that commit hash related to that version?

I can agree on that. What about `2.0.0-pre.6-31-gbdd3fec (dirty)`. The entire point of this PR is to move from just providing commit SHAs to something with more face value. What about the version displayed on the client? Does this apply to the client as well?